server/plugin.go

```diff
@@ -241,8 +241,6 @@ func (p *Plugin) startMeeting(user *model.User, channel *model.Channel, meetingI
 			"meeting_personal":        meetingPersonal,
 			"meeting_topic":           meetingTopic,
 			"from_webhook":            "true",
```

Comment:
Is `from_webhook` still needed given that `override_username` and `override_icon_url` got removed?

Reply:
yes, fair point, probably removing that is going to remove the label "bot" but I think is not needed for this cases where is created directly through user interaction.